sections/02_intro.md

```diff
@@ -34,88 +34,118 @@ learning methods more challenging or less fruitful to apply.
-exobytes of high quality data. Others, like chemistry, are comparatively data poor with well developed domain specific and effective algorithms. These
-differences become consequential and define the most successul approachs. For example, the convergence of lower amounts of data
-and important nuances to the domain might favor lower parameter networks that incorporate domain specific knowledge and fuse data of multiple
-different types. This flexibility, remember, is one of the most striking strengths of neural networks. In the long run,
-it is an open question the most effect strategies will be, but in this time of creative experimenation optimism is justified.
-
-None of these scientific concerns should dampen enthusiasm about deep learning. Rather, because the approaches flexibility,
-there is good reason to believe that carefully defined networks might enbable important scientific advances.
+Given the intrinsic flexibility of deep learning, it is important to consider
+the specific values and goals that are particularly important in scientific
+inquiry.
+
+First, in scientific contexts understanding the patterns in the data may be just
+as important as fitting the data. For this reason, interpretability can be more
+important here than other domains. Scientific work often aims to understand the
+underlying principles behind the data we see, and architectures and techniques
+that expose the non-obvious patterns in the data are particularly important and
+very active area of research `TODO: cite examples from all sections`.
+
+Second, there are important and pressing questions about how to build networks
+that can efficiently represent  the underlying logic of the data. This concern
+of "representability" is important, because it gives insight into the structure
+of scientific data, and when understood can guide the design of more efficient
+and effective networks. For example, one particularly important study
+demonstrates that a simple neural network can very efficiently and accurately
+model (i.e. represent) a theoretically important quantum mechanical system
+[@doi:10.1126/science.aag2302].
+
+Third, science is full of domain expertise, where there are deep traditions of
+thought stretching back decades and even centuries. Deep learning will always be
+in dialogue with this expertise, to understand the key problems, encode the most
+salient prior knowledge, and understand how to judge success or failure. There
+is a great deal of excitement about deep learning, but in most scientific
+corners careful thought needs to be put into bringing deep learning alongside
+existing experts and efforts. `TODO: appropriate comparisons with state of the
+art`
+
+Fourth, data availability and complexity is unevenly distributed across science.
+Some areas of science like genomics and particle physics are swamped in
```

Comment:
Astronomy is the other bug data generator. http://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002195 is a nice paper on big data in biology, c.v. "4-headed beast"